src/libs/actions/Report.js

```diff
@@ -400,7 +434,10 @@ function fetchChatReportsByIDs(chatList, shouldRedirectIfInaccessible = false) {
             Onyx.mergeCollection(ONYXKEYS.COLLECTION.REPORT, simplifiedReports);
 
             // Fetch the personal details if there are any
-            PersonalDetails.getFromReportParticipants(_.values(simplifiedReports));
+            Promise.resolve(PersonalDetails.getFromReportParticipants(_.values(simplifiedReports))).then(
+                details => configureReportNameAndIcon(_.values(simplifiedReports), details),
+            );
```

Comment:
```suggestion
	     const simplifiedReportsList = _.values(simplifiedReports);
            PersonalDetails.getFromReportParticipants(simplifiedReportsList)
            .then(details => configureReportNameAndIcon(simplifiedReportsList, details));
```

Reply:
In this case, we should also return promise in case particpantEmails.length === 0 from PersonalDetails.getFromReportParticipants()
```
  if (participantEmails.length === 0) {
      return Promise.resolve();
  }
```

Reply:
Yeah, that is fine. the interface should be consistent.